aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/utils/S3RequestUtils.java

```diff
@@ -64,9 +83,30 @@ private static String getAccessLevelPrefix(
             @NonNull String identityId
     ) {
         if (accessLevel.equals(StorageAccessLevel.PRIVATE) || accessLevel.equals(StorageAccessLevel.PROTECTED)) {
-            return accessLevel.name().toLowerCase(Locale.US) + "/" + identityId + "/";
+            return accessLevel.name().toLowerCase(Locale.US) + BUCKET_SEPARATOR + identityId;
         } else {
-            return accessLevel.name().toLowerCase(Locale.US) + "/";
+            return accessLevel.name().toLowerCase(Locale.US);
         }
     }
+
+    @NonNull
+    private static String getAccessLevelPrefix(@NonNull StorageOptions options) {
+        StorageAccessLevel accessLevel = options.getAccessLevel();
+        if (accessLevel == null) {
+            throw new RuntimeException(new StorageException(
```

Comment:
Will the run time exception get caught for sure? Or should you just throw a storage exception directly, to be sure/safe? Same for all in this file

Reply:
These utility methods aren't yet used anywhere to be honest. I just remembered that I went through the trouble of abstracting out options to make it easier to test, but I realized that I didn't actually use it anywhere. I think this will be used later on in one of the refactors once we are done with the testing shenanigans, so we'll see exactly how this utility gets used.